mysql/cloudmysql/cloudmysql.go

```diff
@@ -75,8 +75,9 @@ type Params struct {
 	TraceOpts []ocsql.TraceOption
 }
 
-// Open opens a Cloud SQL database.
-func Open(ctx context.Context, certSource proxy.CertSource, params *Params) (*sql.DB, error) {
+// Open opens a Cloud SQL database. The second return value is a Wire cleanup
```

Comment:
Nit: I would leave `Wire` out of the description; if you use this without wire you can still use it.

Reply:
Done.